addons/info/src/panel/empty_panel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const styles = {
+  main: {
+    fontFamily:
+      '-apple-system, ".SFNSText-Regular", "San Francisco", Roboto, "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif',
```

Comment:
Use theme from lib/components, or better yet, don't try to change to another font.

Preferably do not override the font at all.

Reply:
Ok!